environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/model/request/EnvironmentEditRequest.java

```diff
@@ -31,6 +32,9 @@
     @ApiModelProperty(EnvironmentModelDescription.TELEMETRY)
     private TelemetryRequest telemetry;
 
+    @ApiModelProperty(EnvironmentModelDescription.BACKUP)
+    private @Valid BackupRequest backupRequest;
```

Comment:
Sorry for noticing this point just now but I would like to ask to rename the `backupRequest` to simply `backup`. I don't know how big effort is this at the UI side.
Similar renames are commented below.

Reply:
Thanks! Changed it.